pypeit/core/load.py

```diff
@@ -291,8 +285,24 @@ def load_1dspec(fname, exten=None, extract='OPT', objname=None, flux=False):
         if exten < 0:
             msgs.error("Bad input object name: {:s}".format(objname))
 
+    # Keywords for Table
+    rsp_kwargs = {}
+    if flux:
+        rsp_kwargs['flux_tag'] = '{:s}_FLAM'.format(extract)
+        rsp_kwargs['sig_tag'] = '{:s}_FLAM_SIG'.format(extract)
+    else:
+        rsp_kwargs['flux_tag'] = '{:s}_COUNTS'.format(extract)
+        rsp_kwargs['sig_tag'] = '{:s}_COUNTS_SIG'.format(extract)
+
+    # Use the WAVE_GRID (for 2d coadds) if it exists, otherwise use WAVE
+    rsp_kwargs['wave_tag'] = '{:s}_WAVE_GRID'.format(extract)
     # Load
-    spec = XSpectrum1D.from_file(fname, exten=exten, **rsp_kwargs)
+    try:
```

Comment:
The relevant wavelength tag is called WAVE_GRID for 2d coadds, but it is WAVE for normal reductions. This is because 2d coadds have both wavelengths, but regular reductions only have the straight extracted wavelengths since they are not extracted onto a grid. We could discuss if there is a better notation here. But for 2-d coadds you always want the WAVE_GRID hence the try except. If the tag is there it will be used since it is the preferred wavelength.